src/arrayMethodSort.js

```diff
@@ -1,11 +1,42 @@
+/* eslint-disable no-unused-expressions */
 'use strict';
 
 /**
  * Implement method Sort
  */
+
+const compareString = (a, b) => {
+  const firstString = a.toString();
+  const secondString = b.toString();
+
+  switch (true) {
+    case firstString > secondString:
+      return 1;
+    case firstString < secondString:
+      return 0;
+    default:
+      return -1;
+  }
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareString) {
+    let count;
+
+    for (let i = 1; i < this.length; i++) {
```

Comment:
can you use the array method instead of a simple loop?

Reply:
I think every array methods are prohibited  in this task)